Source/WebKit/WebProcess/Network/WebLoaderStrategy.cpp

```diff
@@ -364,6 +364,8 @@ static void addParametersShared(const LocalFrame* frame, NetworkResourceLoadPara
 
     if (isMainFrameNavigation)
         parameters.linkPreconnectEarlyHintsEnabled = mainFrame->settings().linkPreconnectEarlyHintsEnabled();
+
+    parameters.shouldIgnoreIframeEmbeddingProtections = mainFrame->settings().ignoreIframeEmbeddingProtectionsEnabled();
```

Comment:
Would it be more desirable for the UI process to tell the network process this is enabled rather than the web process telling the network process?

You can use the option sharedPreferenceForWebProcess in UnifiedWebPreferences for this. This will make the value generated in SharedPreferencesForWebProcess which is accessible in the network process in the NetworkConnectionToWebProcess.

Reply:
truth